### PR TITLE
Add jsonpathly implementation

### PR DESCRIPTION
**Pull request by atamano**
Add javascript jsonpathly implementation

### CODE REVIEW

File: src/compile_bug_reports.sh [hunk 1]
```diff
@@ -143,7 +143,9 @@ process_implementation() {
     header
 
     while IFS= read -r query; do
-        failing_query "$implementation" "$query"
+	if [ -n "${query}" ]; then
```
Comment (atamano) [inline src/compile_bug_reports.sh:146]: @cburgmer added this condition because sometimes "query" was empty

Comment (cburgmer) [reply src/compile_bug_reports.sh:146]: If I understand correctly, this is about `$(all_incorrect_queries)` returning nothing?

**Comment by cburgmer** [comment]
I re-ran locally, and found some more changes. 
Could you have a look at https://github.com/cburgmer/json-path-comparison/commit/24f65cec800f8ffde5faad9e82f533793192204e#diff-243f00a070d49969a2b1fbc7ab9be6b8917edebb0f8807a1ddbc3e70e9f14ba4 and see why locally I got different output for two queries against jsonpathly?

**Comment by atamano** [comment]
> I re-ran locally, and found some more changes. 
> Could you have a look at https://github.com/cburgmer/json-path-comparison/commit/24f65cec800f8ffde5faad9e82f533793192204e#diff-243f00a070d49969a2b1fbc7ab9be6b8917edebb0f8807a1ddbc3e70e9f14ba4 and see why locally I got different output for two queries against jsonpathly?

Your generation is right, I had made little changes to my index.js after generating and forgot to regenerate

File: src/compile_bug_reports.sh (hunk 1 above)
Comment (atamano) [reply src/compile_bug_reports.sh:146]: sometimes "query" is empty, I haven't explored more about it

**Pull request merged**